dashboard/app/models/census/census_submission.rb

```diff
@@ -86,4 +86,28 @@ class Census::CensusSubmission < ApplicationRecord
   validates :submitter_email_address, length: {maximum: 255}
   validates :submitter_name, length: {maximum: 255}
   validates :topic_other_description, length: {maximum: 255}
+
+  def inaccuracy_review_data
+    school_infos.map do |si|
+      next unless si.school
+      current_summary = si.school.census_summaries.select {|s| s.school_year == school_year}.first
```

Comment:
use `find` instead of `select.first` so we don't evaluate the select block unnecessarily.

Reply:
Done.